plugins/polio/js/src/domains/Campaigns/Scope/Scopes/DistrictScopeTable.tsx

```diff
@@ -271,7 +261,7 @@ export const DistrictScopeTable: FunctionComponent<Props> = ({
                                                 }
                                             />
                                         )}
-                                        {isShapeInScope && (
+                                        {shape.vaccineName && (
```

Comment:
shape.vaccineName is never true for non polio scopes, so the toggle works, but the UI always displays the "add" icon